doc/doc-primitives.saty

```diff
@@ -59,9 +63,129 @@ document (|
       どのフォントで組まれるかや「空白がどの程度伸縮できるか」「どこで改ページしてよいか」といった
       情報も保持している．
     }
+    +type (tMATH) {
+      数式を扱う型．
+    }
+    +type (tMATHCLS) {
+      数式のスペーシングのために必要な“どの種類の数学記号なのか”を表す型．
+      \code{MathOrd}（通常）・\code{MathBin}（2項演算子）・\code{MathRel}（関係子）・
+      \code{MathOp}（作用子）・\code{MathPunct}（パンクチュエーション）・
+      \code{MathOpen}（開き括弧類）・\code{MathClose}（閉じ括弧類）・\code{MathPrefix}（接頭類）・
+      \code{MathInner}（部分式）の9つのデータがある代数的データ型である．
+    }
+    +type (tMCCLS) {
+      数式中に書かれている文字をどの文字クラスに基づいてグリフに変換するときに使用する型．
```

Comment:
> …文字をどの文字クラスに基づいてグリフに変換するときに使用する型．

does not make much sense.

Do you mean something like

> …文字を文字クラスに基づいてグリフに変換するときに使用する型．

BTW, shouldn't this 文字クラス be renamed with some other term because 文字クラス ([character class](https://unicode.org/glossary/#character_class)) has a different meaning in the context of Unicode.

I don't know what we should call it.  Something like 数式用文字書体?